doc/source/dev/roadmap-detailed.rst

```diff
@@ -415,46 +418,36 @@ The following improvements will help SciPy better serve this role.
     - nested ANOVA
     - analysis of covariance (ANCOVA)
 
-- Where appropriate, include confidence intervals for the statistic in the
-  results of any statistical test.
 - Add additional tools for meta-analysis; currently we have just `combine_pvalues`.
 - Enhance the `fit` method of the continuous probability distributions:
 
   - Expand the options for fitting to include:
 
-    - method of moments
     - maximal product spacings
     - method of L-moments / probability weighted moments
 
   - Include measures of goodness-of-fit in the results
-  - Handle censored data
+  - Handle censored data (e.g. merge `gh-13699 <https://github.com/scipy/scipy/pull/13699>`__)
 
 - Implement additional widely used continuous and discrete probability
   distributions:
 
-  - noncentral hypergeometric distribution (both Fisher's and Wallenius')
-  - negative hypergeometric distribution
-  - multivariate hypergeometric distribution
   - multivariate t distribution
   - mixture distributions
 
 - Improve the core calculations provided by SciPy's probability distributions
   so they can robustly handle wide ranges of parameter values.  Specifically,
   replace many of the PDF and CDF methods from the Fortran library CDFLIB
   used in scipy.special with better code, perhaps ported from the Boost C++
-  library.
+  library. (e.g. merge `gh-13328 <https://github.com/scipy/scipy/pull/13328>`__))
 
 In addition, we should:
 
 - Continue work on making the function signatures of ``stats`` and
   ``stats.mstats`` more consistent, and add tests to ensure that that
   remains the case.
-- Return ``Bunch`` objects from functions that now return many values, and for
-  functions for which extra return values are desired (see
-  `gh-3665 <https://github.com/scipy/scipy/issues/3665>`__).
-- Improve statistical tests (p-value calculation, alternative hypothesis), for
-  example implement an exact two-sided KS test (see
-  `gh-8341 <https://github.com/scipy/scipy/issues/8341>`__) or a one-sided
-  Wilcoxon test (see `gh-9046 <https://github.com/scipy/scipy/issues/9046>`__).
-- Address the various issues regarding ``stats.mannwhitneyu``, and pick up the
-  stalled PR in `gh-4933 <https://github.com/scipy/scipy/pull/4933>`__.
+- Improve statistical tests: consistently provide options for one- and
+  two-sided alternative hypotheses where applicable, return confidence
+  intervals for the test statistic, and implement exact p-value calculations -
+  considering the possibility of ties - where computationally feasible.
```

Comment:
We've added alternative hypotheses for many functions (summary issue gh-12506).

Confidence intervals for `binomtest` were added in gh-12603. They'll be added for pearsonr in gh-12609, and bootstrap confidence intervals can be calculated for any statistic using gh-13371. The latter could be used to add a confidence interval method to the return object of existing tests, or it can be used as a standalone function.

After gh-4824, it is easy to generalize the computation of randomized and exact permutation tests, and I'll submit a PR for that after some other PRs make it in (e.g. gh-13661). (Of course there will still be room for faster exact null distribution calculations e.g. recursion.) I've also written a class that brings everything together. With that, adding new statistics tests with all these features (and `nan_policy` support, `axis` argument, etc.) is mostly a matter of defining the test statistic and asymptotic null distribution. Just like `rv_continuous` has default methods to fall back on if only `_pdf` is defined, much of a statistical test calculation can be taken care of automatically (in many cases). I'll submit a PR for that in a bit, too, but again I'd like to clear out the backlog a bit first.